keycloak/keycloak-admin-ui
# Admin events rep dialog

## Motivation
1. Mimic representation dialog for now until #1132 is delivered -> https://marvelapp.com/prototype/7ca9dhf/screen/73036562
2. Add scaffolding for the Client Policies tab under Realm Settings ->https://marvelapp.com/prototype/h11728e/screen/81529211

## Verification Steps
To verify representation dialog: 

1. Go to /events/adminEvents, select <more icon>, select representation option.
2. Verify that the dialog is as per designs and is displaying the hardcoded object.

To verify scaffolding for the Client Policies tab under Realm Settings:
1. Go to /realm-settings/clientPolicies and verify that Profiles and Policies tabs are displaying with each displaying help text on "?" selection. The help text is as per the old admin console.

## Checklist:
